content/learningpaths/effective-efficient-kubernetes-debugging/kubernetes-debugging-workflow-and-techniques.md

```diff
@@ -67,216 +67,210 @@ flag will allow the output to be watched.
 netshoot container and exec into it to perform tests as follows,
 
 ```sh
 kubectl run netshoot --image=nicolaka/netshoot && \
 kubectl exec -it -n netshoot "--" sh -c "clear; (bash || ash || sh)"
 ```
 
-However, note that this is a new pod and itself would not have access to a
-target pod's filesystem.  This approach is useful for debugging aspects such as
+However, this is a new pod that does not have access to a
+target pod's filesystem. This approach is useful for debugging aspects such as
 network connectivity. A deep dive of these aspects as classes is explored in
 the next learning path on Heuristic Approach.
 
-### Using ephemeral debug containers
+### Using Ephemeral Debug Containers
 
-As realized in the previous section, using `kubectl exec` may be insufficient
-due to the target container having no shell or that the container has already
-crashed and is inaccessible.  One could overcome this using a different image
-deployed via `kubectl run`, however, this would be a different pod that is not
-sharing the process namespace of the pod of interest for debugging, meaning that
-one cannot access the process details or its filesystem in a different pod. This
-limitation can be alleviated using debug ephemeral containers.
+As discussed in the previous section, using `kubectl exec` may be insufficient
+due to the target container having no shell, or if the container has already
+crashed and is inaccessible. To overcome this, use a different image
+deployed via `kubectl run`. This should be a different pod that is not sharing 
+the process namespace of the pod of interest for debugging. This means you 
+cannot access the process details or its filesystem in a different pod. To alleviate
+this limitation, use debug ephemeral containers.
 
 Introduced as an alpha in Kubernetes 1.16 and currently as a [beta in Kubernetes
 1.23](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#ephemeralcontainer-v1-core),
 ephemeral containers are helpful for troubleshooting issues. By default, the
 `EphemeralContainer` feature-gate [is
 enabled](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
-as a beta in Kubernetes 1.23. Otherwise, the feature-gate will need to be
-enabled first before it can be used. This would require enabling the . The documentation for [debug
-containers](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-running-pod/#ephemeral-container),
-demonstrate with examples of how to use `debug containers` but for brevity and
-completeness sake, this is included below together with content aligning to this
+as a beta in Kubernetes 1.23. Otherwise, you must enable the feature-gate first before it can be used. 
+The documentation for [debug containers](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-running-pod/#ephemeral-container)
+use examples to demonstrate how to use `debug containers`. It is included ias follows, along with content aligning to this
```

Comment:
`It is included ias follows`

->

`It is included as follows`

Reply:
fixed.